src/main/java/frc/robot/commands/climber/ManualControlClimber.java

```diff
@@ -24,7 +24,7 @@ public void end(boolean interrupted) {
     @Override
     public void execute() {
         double value = supplier.getAsDouble();
-        if (Math.abs(value) > .2) {
+        if (Math.abs(value) > .2 && climber.getEncoderPosition() < 80 && climber.isExtendedLimitSwitchNotPressed()) {
```

Comment:
1 - no need to check the limit switch because it is connected directly to the motor controller.
2 - this is probably not the best place to put this code, there are 2 encoders, one for each arm, and this code will not work. Can you move the logic to the subsystem setSpeed method?